src/MultiFloats.jl

```diff
@@ -495,10 +495,19 @@ function Base.log(x::MultiFloat{T,N}) where {T,N}
     return y
 end
 
+function Base.log1p(x::MultiFloat{T,N}) where {T,N}
+    y = x+1
+    return Base.log(y)
+end
+
+function Base.expm1(x::MultiFloat{T,N}) where {T,N}
+    return Base.exp(x)-1
```

Comment:
This could be written as
```
t = tanh(x/2)
return 2*t/(1-t)
```

Reply:
This identity will give a relatively efficient implementation that should be accurate to 3 ULP or so.

Reply:
Done